src/controllers/handlers/comms-scene-handler.ts

```diff
@@ -28,18 +28,12 @@ export async function commsSceneHandler(
     room = `preview-${identity}`
 
     forPreview = true
-    permissions = {
-      cast: [],
-      mute: []
-    }
   } else if (realmName.endsWith('.eth')) {
-    permissions = await sceneFetcher.fetchWorldPermissions(realmName)
     room = livekit.getWorldRoomName(realmName)
   } else {
     if (!sceneId) {
       throw new UnauthorizedError('Access denied, invalid signed-fetch request, no sceneId')
```

Comment:
We should return a 400 when the request is done with invalid parameters, what do you think?